imports/components/players/audio/audio.Scrubber.js

```diff
@@ -50,19 +55,48 @@ export default class AudioScrubber extends Component {
 
     if(typeof percent === "number") {
       this.seek(percent);
-    }
+    };
 
     setTimeout(() => {
+      // Set states back to intial values so that time updates,
+      // and click events will happen appropriatly.
+      // Timeout is a full second as a shorter timeout did not
+      // set this states properly.
       this.setState({
-        lastPercent: null,
+        scrubbing: false,
```

Comment:
@edolyne we should add a check to ensure the component is mounted on this. Or add the timeout to the instance and clear it on umount